mass/core/channel_group.py

```diff
@@ -631,9 +631,9 @@ def calc_external_trigger_timing(self, forceNew=False):
                    ("subframes_from_nearest_external_trigger" in ds.hdf5_group) and (not forceNew):
                     continue
 
-                subframes_after_last, subframes_until_next = \
-                    mass.core.analysis_algorithms.nearest_arrivals(ds.p_subframecount[:],
-                                                                   external_trigger_subframe_count)
```

Comment:
https://github.com/usnistgov/mass/blob/df0e1254a8ebf260d7f6c045489c3fbf72451657/mass/core/channel.py#L850-L864

There used to be a property `external_trigger_rowcount` in `Channel` that loaded the file automatically. I guess it got deleted and nobody noticed? Anyway this is cleaner and it's good to add a test.

Reply:
Well, are you getting mixed up with the OFF object? The OFF channel's object turned into `external_trigger_subframe_count`, which is still there.

Reply:
No I'm not getting mixed up with off, I provided a perma link to an older version of `mass/core/channel.py` showing the property that used to exist. I think that must have been deleted when we changed stuff to `subframe`, thus breaking this code. This is cleaner as it only loads the `external_trigger.bin` file once.

Reply:
Oops. I got confused about the `external_trigger_rowcount` vs the one that starts with an underscore.

Okay, I think your comment helps to explain it: I removed the double-reading of the bin file when working with the J-PARC data in 2024, and I hadn't realized until today that doing so broke this until_next/after_last code.
Yes, let's go with the new, one-read way.